src/main/resources/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerViewImpl.ui.xml

```diff
@@ -35,6 +35,9 @@
         <bh:Div ui:field="createVersionJobTrackingWidgetContainer" />
       </b:ModalBody>
     </b:Modal>
-    <bh:Div ui:field="extraWidgetsContainer" />
+    <bh:Div
+      ui:field="extraWidgetsContainer"
+      addStyleNames="extraWidgetsContainer"
```

Comment:
remove

Reply:
I think this had to do with controlling where Bootstrap modals were inserted into the DOM.  You can either explicitly add them to the DOM like this, or let Bootstrap create/add automatically (which ends up inserting up the hierarchy, maybe in the body?)

Reply:
Sorry, this was a note to myself to remove the `addStyleNames`.